ros_buildfarm/templates/devel/devel_task.Dockerfile.em

```diff
@@ -55,14 +57,14 @@ RUN echo "@today_str"
 USER buildfarm
 ENTRYPOINT ["sh", "-c"]
 @{
+cmd = 'PATH=/usr/lib/ccache:$PATH' + \
```

Comment:
Good catch. I made all the command extensions consistently prefixed with whitespace to be consistent with the debbuild job, and internally consistent.